src/routes/community/news/NewsStyle.js

```diff
@@ -1,4 +1,136 @@
 import styled from 'styled-components';
 import palette from '@styles/theme';
 
-export const News = styled.div``;
+export const NewsPage = styled.div`
+  display: flex;
+  justify-content: center;
+  gap: 125px;
+  padding-top: 70px;
+
+  @media (min-width: 1280px) {
+    padding-top: 192px;
+  }
+
+  @media (max-width: 767px) {
+    padding-top: 30px;
+  }
+`;
+
+export const News = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const ItemGroup = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+`;
+
+export const Title = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: start;
+  color: ${palette.mainNavy.navy100};
+
+  font-size: 36px;
+  font-weight: 600;
+  line-height: 150%;
+  letter-spacing: -0.9px;
+
+  margin-bottom: 30px;
+
+  @media (max-width: 767px) {
+    font-size: 16px;
+    line-height: 130%;
+    letter-spacing: -0.4px;
+  }
+`;
+
+export const Group = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const SocialBox = styled.a`
+  display: flex;
+  padding: 10px 20px;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+
+  border-radius: 20px;
+  background: ${palette.background.white};
+
+  box-shadow: 0 0 15px 0 rgba(18, 32, 102, 0.1);
```

Comment:
```suggestion
  box-shadow: 0 0 15px 0 ${palette.mainNavy.navy10};
```
해당 값도 색상이 팔레트에 있습니다 참고해주세요!